modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -370,7 +368,11 @@ export default {
           return false;
         }
       }
-      this.editing = this.items.find(item => item._id === id);
+      if (!item?._edit) {
+        this.editing = null;
+        return true;
```

Comment:
Not returning `true` here set off a cascade of bugs that distracted me for half an hour 😆